test_scripts/WebEngine/WebEngineProjection/001_Registration.lua

```diff
@@ -23,20 +23,27 @@
 local common = require('test_scripts/WebEngine/commonWebEngine')
 
 --[[ Local Variables ]]
+local appSessionId = 1
 local appHMIType = "WEB_VIEW"
 
 --[[ General configuration parameters ]]
 config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
 config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
 config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
 
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
-common.Step("Start SDL, HMI, connect Mobile", common.start)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
```

Comment:
@ydementieiev I would recommend to change it to `Update AppHMIType in preloaded policy table` as in PT there is no such field like `appHMIType` but `AppHMIType`  is valid one. This is related to all scripts where you add AppHMIType to PT.

Reply:
@AByzhynar thank you, [changed](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/abede64e2119bad9c40c14ecd9e5ef9e03ce0228)